### PR TITLE
added polish translation

### PR DESCRIPTION
**Pull request by tkuska**
### Changelog

```markdown
### Added
- Added Polish translation in `SonataDoctrinePHPCRAdmin.pl.xliff`
```

### Subject

Added Polish translation

### CODE REVIEW

**Comment by OskarStark** [comment]
Thank you @tkuska

**Comment by OskarStark** [comment]
Can you please fix travis?

```diff
-<?xml version="1.0"?>
-<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
-    <file source-language="pl" datatype="plaintext" original="" >
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file source-language="pl" datatype="plaintext" original="">
         <body>
             <trans-unit id="content_tree">
                 <source>content_tree</source>
@@ -33,7 +33,7 @@
             <trans-unit id="label_type_contains_words">
                 <source>label_type_contains_words</source>
                 <target>zawiera słowa</target>
-            </trans-unit>            
+            </trans-unit>
         </body>
-     </file>
+    </file>
 </xliff>
```

**Comment by tkuska** [comment]
can you help me with reason of this issue?
I'm new with travis CI and have no idea whats goes wrong..

**Comment by dbu** [comment]
it seems that the formatting is wrong. check for trainling whitespaces and alignment. and the header line seems to be ordered differently and the linter complains about that.

**Comment by OskarStark** [comment]
> it seems that the formatting is wrong. check for trainling whitespaces and alignment. and the header line seems to be ordered differently and the linter complains about that.

that is true, plus in the first line for example, this is added: `encoding="UTF-8"`

please check it line by line, and make it look like the green version, or apply this as as a patch directly

**Comment by OskarStark** [comment]
Looks good, except the EOF ;-)

**Comment by OskarStark** [comment]
Awesome, now please squash your commit into one!

Afterwards we're good to go! 👍

**Comment by tkuska** [comment]
did not know it's relevant, now should be better
thanks for help :)

**Comment by OskarStark** [comment]
If you need help with `squashing`, please let me know! 👍

File: Resources/translations/SonataDoctrinePHPCRAdmin.pl.xliff [hunk 1]
```diff
@@ -33,7 +33,7 @@
             <trans-unit id="label_type_contains_words">
                 <source>label_type_contains_words</source>
                 <target>zawiera słowa</target>
-            </trans-unit>            
+            </trans-unit>
         </body>
-     </file>
-</xliff>
+    </file>
+</xliff>
```
Comment (greg0ire) [inline Resources/translations/SonataDoctrinePHPCRAdmin.pl.xliff]: Please don't change line 39 and configure your editor to keep a newline at end of file.

**Comment by Soullivaneuh** [comment]
@tkuska One last thing to do: https://travis-ci.org/sonata-project/SonataDoctrinePhpcrAdminBundle/jobs/137501512#L244

**Comment by tkuska** [comment]
@Soullivaneuh yes i've seen this and i will fix it.
@OskarStark i would need help with squashing, tried to do this in sourcetree but i failed

**Comment by dbu** [comment]
we can let github squash during the merge operation - anything wrong with that? i stopped asking people to squash since they added that feature.

**Comment by OskarStark** [comment]
> we can let github squash during the merge operation - anything wrong with that? i stopped asking people to squash since they added that feature.

but then we have no merge commit

/cc @greg0ire @Soullivaneuh

**Comment by Soullivaneuh** [comment]
>> we can let github squash during the merge operation - anything wrong with that? i stopped asking
>> people to squash since they added that feature.

> but then we have no merge commit

Wrong and already discussed. You have a merge commit, it will be the commit itself. :+1: 

@dbu you can if the PR contains only one valid commit.

After that, like @greg0ire said, if people can learn how to squash commit thanks to our comments, it's a plus! :+1:

**Comment by greg0ire** [comment]
> we can let github squash during the merge operation - anything wrong with that? i stopped asking people to squash since they added that feature.

It's ok for commits like this 

- do something
- fix stupid mistake in first commits
- add documentation

But it should not be used IMO for commits like this : 

- do some unrelated cleanup unrelated but that happens to be on the same file / line
- work happily on the feature now that the file is clean

**Comment by OskarStark** [comment]
> Wrong and already discussed. You have a merge commit, it will be the commit itself. 👍

ofc, but not a real merge commit, thats what i meant!

**Comment by OskarStark** [comment]
@tkuska do `git rebase -i HEAD~4` and leave the first commit as it is!

replace the word `pick` by an `f` and save the file!

**Comment by Soullivaneuh** [comment]
> ofc, but not a real merge commit, thats what i meant!

And why this is important? The PR marked as merged and the history is clearer.

Even if the commits are squashed by the user onto one, the merge by squash is the way to use at the end! :+1:

**Comment by OskarStark** [comment]
> And why this is important? The PR marked as merged and the history is clearer.

i thought this was our decision when we talked about the StyleCi PR's by SonataCI

**Comment by Soullivaneuh** [comment]
> i thought this was our decision when we talked about the StyleCi PR's by SonataCI

Not sure too understand... StyleCI and SonataCI PR are merged by squash.

The classic merge is done only if the PR contains multiple **valid** commits.

**Comment by Soullivaneuh** [comment]
Anyway, the PR is green, let's merge.

**Comment by tkuska** [comment]
thanks for help guys

**Comment by Soullivaneuh** [comment]
Thank you for your contribution. :+1:

**Pull request merged**